crates/tx5-connection/src/hub.rs

```diff
@@ -239,14 +239,15 @@ impl Hub {
                         }
                     }
                     HubCmd::Disconnect(pub_key) => {
-                        // disconnect from a remote peer
+                        // Remove remote peer from hub map
```

Comment:
First remove the peer from the hub map, then call close_peer.

If `DynBackEp::connect` is called before this `close_peer` future resolves, then that connection attempt would fail with an error "already connected".

Since we are ignoring the result from `close_peer` here anyway, this does not change the logic.